powermon/commands/abstractCommand.py

```diff
@@ -6,22 +6,24 @@
 
 log = logging.getLogger("Command")
 
+
 class CommandType(LowercaseStrEnum):
     POLL = auto()
 
+
 class AbstractCommand(ABC):
```

Comment:
Too many things called command, seeing as you are moving away from the schedule idea you can delete this class and stick with your new type of command.

Reply:
have started to tidy up things

tried to get the api to run, but it was throwing an error before starting - do you have your start process? (I put in the wiki what steps I was taking)

Reply:
In the docker-compose.development.yaml file it defines how to run the API. 

One thing that could be causing the issue is I am using docker with the compose plugin. Which is slightly different than the docker-compose standalone. I will switch to the docker-compose standalone since that's what you are already using.

I launch it with the makefile command you setup, docker-powermon-dev-up. You will notice that I removed the hyphen from that command since it wasn't working for me.